tools/sendinblueAPIPlayground.py

```diff
@@ -0,0 +1,24 @@
+import os
+import requests
+from pprint import pprint
+
+listes_ecosante_folder_name = 'listes_ecosante'
+
+# Start with
+# SENDINBLUE_API_KEY=<key> python3 tools/sendinblueAPIPlayground.py 
+
+if __name__ == "__main__":
+    api_key = os.environ['SENDINBLUE_API_KEY']
```

Comment:
You may want to use `os.getenv` it will return `None` if the env var doesn't exist